### DISCUSSION TITLE
How to create float window on top when using fullscreen mode?

### DISCUSSION THREAD

**Discussion by OriLiMu** [qtile/qtile#4479] [Q&A]
When I am in fullscreen mode, then I press <kbd>Mod</kbd> + <kbd>Return</kbd> twice. 
First time it will generated a fullscreen terminal, then it will generate a float terminal on top. 
Now I don't want the first terminal just give me the second float terminal will be fine.
How to do it?

**Comment by jrwrigh** [comment]
As in you want to kill the first terminal and replace it with the second terminal?

You'd have to create a custom function that is triggered off of <kbd>Mod</kbd> + <kbd>Return</kbd> that will check if you're in full-screen mode, see if there's a terminal currently open, then remove that terminal and open a new one.

Possibly dumb question, but maybe a simpler solution; why not just run `clear` in the first terminal to clear the screen? I totally get wanting to use a more familiar interface to do that, but `clear` seems like the much simpler solution here.
